stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/composer/MessageComposerController.kt

```diff
@@ -442,6 +442,19 @@ public class MessageComposerController(
         handleValidationErrors()
     }
 
+    /**
+     * Replaces the previously selected attachments with the newly selected attachments from the attachment
+     * picker. These will be shown in the UI, within the composer component. We upload and send these
+     * attachments once the user taps on the send button.
+     *
+     * @param attachments The attachments to store and show in the composer.
+     */
+    public fun setSelectedAttachments(attachments: List<Attachment>) {
```

Comment:
There's a difference in how attachment pickers for Compose and UI Components behave. The former adds attachments to the existing selection, and the latter replaces the currently selected attachments. Adding this method to be consistent with the existing behavior in UI Components.